nipype/interfaces/io.py

```diff
@@ -2863,3 +2864,33 @@ def _list_outputs(self):
 
     def _add_output_traits(self, base):
         return add_traits(base, list(self.inputs.output_query.keys()))
+
+
+class ExportFileInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, desc='Input file name')
+    out_file = File(exists=False, desc='Output file name')
+    check_extension = traits.Bool(False, desc='Ensure that the input and output file extensions match')
+    clobber = traits.Bool(False, desc='Permit overwriting existing files')
+
+
+class ExportFileOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='Output file name')
+
+
+class ExportFile(BaseInterface):
+    input_spec = ExportFileInputSpec
+    output_spec = ExportFileOutputSpec
+
+    def _run_interface(self, runtime):
+        if not self.inputs.clobber and op.exists(self.inputs.out_file):
+            raise FileExistsError(errno.EEXIST, f'File {self.inputs.out_file} exists')
```

Comment:
f-strings are 3.6+, we support down to 2.7 and 3.4 for now, and will move to 3.5+ in a couple releases.

```suggestion
            raise FileExistsError(errno.EEXIST, 'File %s exists' % self.inputs.out_file)
```